internal/search/query/fields.go

```diff
@@ -0,0 +1,41 @@
+package query
+
+type none struct{}
+
+var empty = none{}
```

Comment:
Will `none` be used elsewhere? Maybe just this (if it compiles):
```suggestion
var empty = struct{}{}
```

Reply:
works for me